pgsmo/objects/table_objects/constraint_exclusion/+default/update.sql

```diff
@@ -8,21 +8,25 @@
 {% if data %}
 {# ==== To update exclusion constraint name ==== #}
 {% if data.name != o_data.name %}
-ALTER TABLE {{ conn|qtIdent(data.schema, data.table) }}
+ALTER TABLE IF EXISTS {{ conn|qtIdent(data.schema, data.table) }}
     RENAME CONSTRAINT {{ conn|qtIdent(o_data.name) }} TO {{ conn|qtIdent(data.name) }};
 {% endif %}
 {# ==== To update exclusion constraint tablespace ==== #}
 {% if data.spcname and data.spcname != o_data.spcname %}
-ALTER INDEX {{ conn|qtIdent(data.schema, data.name) }}
+ALTER INDEX IF EXISTS {{ conn|qtIdent(data.schema, data.name) }}
     SET TABLESPACE {{ conn|qtIdent(data.spcname) }};
 {% endif %}
 {% if data.fillfactor and data.fillfactor != o_data.fillfactor %}
-ALTER INDEX {{ conn|qtIdent(data.schema, data.name) }}
+ALTER INDEX IF EXISTS {{ conn|qtIdent(data.schema, data.name) }}
     SET (FILLFACTOR={{ data.fillfactor }});
 {% endif %}
+{% if data.fillfactor == "" and data.fillfactor != o_data.fillfactor %}
+ALTER INDEX IF EXISTS {{ conn|qtIdent(data.schema, data.name) }}
+    RESET (FILLFACTOR);
+{% endif %}
 {# ==== To update exclusion constraint comments ==== #}
 {% if data.comment is defined and data.comment != o_data.comment %}
 COMMENT ON CONSTRAINT {{ conn|qtIdent(data.name) }} ON {{ conn|qtIdent(data.schema, data.table) }}
-    IS {{ data.comment }};
+    IS {{ data.comment|qtLiteral(conn) }};
```

Comment:
Just wondering, a lot of the qtLiteral was taken out during the psycopg bump up, why was it determined its needed back in some files?

Reply:
added back in the qt_literal filter and added the conn variable when rendering these templates so qtLiteral in the template files should no longer break the scripting!